### ISSUE TITLE
Error, DuplicateFileException: Duplicate files copied in APK

### ISSUE THREAD

**Issue by chrisgonzgonz** [SiftScience/sift-android#19]
I've been getting this error when attempting the standard application integration using gradle v 2.2.2:

```
Error:Execution failed for task ':app:transformResourcesWithMergeJavaResForAppDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.packaging.DuplicateFileException: Duplicate files copied in APK META-INF/LICENSE
  	File1: ~/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.3.3/7d8c5d79cc99995e21e6f955857312d8409f02a1/jackson-core-2.3.3.jar
  	File2: ~/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.3.3/1248c5f768133f45da1691aae19bdc2dfd0b17b0/jackson-annotations-2.3.3.jar
  	File3: ~/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.3.3/63b77400b5f1cf83a81823562c48d3120ef5518e/jackson-databind-2.3.3.jar
```

My `packagingOptions` are below:

```
packagingOptions {
    exclude 'META-INF/DEPENDENCIES.txt'
    exclude 'META-INF/LICENSE.txt'
    exclude 'META-INF/NOTICE.txt'
    exclude 'META-INF/NOTICE'
    exclude 'META-INF/LICENSE'
    exclude 'META-INF/DEPENDENCIES'
    exclude 'META-INF/notice.txt'
    exclude 'META-INF/license.txt'
    exclude 'META-INF/dependencies.txt'
    exclude 'META-INF/LGPL2.1'  
}
```
Not sure what is causing this as the "duplicate" `META-INF/LICENSE` file is being excluded.

Thanks in advance for any guidance!

**Comment by garylee1** [comment]
Thanks for the report @chrisgonzgonz – does this happen with the latest SDK version 0.9.7? Does the gradle update seem to be the culprit?

**Comment by chrisgonzgonz** [comment]
Hi Gary, no problem. This is happening with SDK v 0.9.7 and gradle v 2.2.2. 

Should i be using a different version of the SDK with this version of gradle?

**Comment by garylee1** [comment]
Ah, no just trying to pinpoint the cause. Are you able to build this with a higher gradle version? For testing, we're currently using `3.3`.

**Comment by chrisgonzgonz** [comment]
Unfortunately we're not able to do that, any gradle version `>= 3.0` is breaking for us.

**Comment by garylee1** [comment]
Have you tried using some version of:

```
packagingOptions {
    pickFirst 'META-INF/LICENSE.txt'
}
```

Since it looks like the duplicated files are coming from the same library (Jackson), this should also be okay.

**Comment by chrisgonzgonz** [comment]
Thanks @garylee1 for the suggestion, I've tried that as well. The same failure occurs.

**Comment by garylee1** [comment]
this will be addressed by our upcoming release

**Comment by chrisgonzgonz** [comment]
thank you @garylee1

**Comment by garylee1** [comment]
this has been fixed in 0.9.9

**Comment by garylee1** [comment]
please hold off – we're fixing a small issue related to deserializing archived configs. the fix will be released soon with 0.9.10.

**Comment by chrisgonzgonz** [comment]
great thank you @garylee1

**Comment by garylee1** [comment]
0.9.10 has been released – please skip over 0.9.9
